neoexchange/core/views.py

```diff
@@ -3914,20 +3916,17 @@ def get_lc_plot(body, data):
     """Plot all lightcurve data for given source.
     """
 
-    base_dir = os.path.join(settings.DATA_ROOT, 'Reduction')
-    obj_name = sanitize_object_name(body.current_name())
-    datadir = os.path.join(base_dir, obj_name)
-    filenames = search(datadir, '.*.ALCDEF.txt')
     if data.get('period', None):
         period = data['period']
     else:
         period = 1.0
 
     meta_list = []
     lc_list = []
-    if filenames:
-        for file in filenames:
-            meta_list, lc_list = import_alcdef(os.path.join(datadir, file), meta_list, lc_list)
+    dataproducts = DataProduct.body_objects.filter(filetype=DataProduct.ALCDEF_TXT)
```

Comment:
This code seems to recover ALL `DataProducts` with an ALCDEF `filetype` and a body `content_type` when we actually want only `DataProducts` with an ALCDEF `filetype` connected to `body`. Probably through a variety of superblocks.